packages/css-data/bin/syntaxes.ts

```diff
@@ -0,0 +1,32 @@
+export const propertiesAndFunctionsSyntax = [
```

Comment:
@TrySound where can we put this file? I feel like bin is a bit bad for this type of data, wdyt?

Reply:
🤷

Reply:
I think it belongs to src, not bin, just not sure if we should have a separate file specifically for this or put it into custom-data.ts

Reply:
propertiesAndFunctionsSyntax is not descriptive enough imho, in 2 month I wouldn't know what that is and it needs a comment

Reply:
effectively this is a list of long-hands that we wish existed but they don't, right?

so its something like: customLonghandPropertyNames

Reply:
I think it's better to move it under the `custom-data.ts` file. Instead of a new way, And the name `customLonghandPropertyNames` is self explanatory too 👍